flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontend.java

```diff
@@ -76,13 +74,13 @@ static CliExecutor createExecutor(CommandLine commandLine) throws Exception {
                     "Missing pipeline definition file path in arguments. ");
         }
 
-        // Take the first unparsed argument as the pipeline definition file
-        Path pipelineDefPath = Paths.get(unparsedArgs.get(0));
-        if (!Files.exists(pipelineDefPath)) {
-            throw new FileNotFoundException(
-                    String.format("Cannot find pipeline definition file \"%s\"", pipelineDefPath));
+        Path pipelineDefPath = null;
+        // If Running cdc job,need to set pipeline path
+        if (!ConfigurationUtils.isDeploymentMode(commandLine)) {
+            // Take the first unparsed argument as the pipeline definition file
+            pipelineDefPath = Paths.get(unparsedArgs.get(0));
+            LOG.info("Real Path pipelineDefPath {}", pipelineDefPath);
```

Comment:
We still need to verify `Files.exists(pipelineDefPath)` in `local` and `remote` mode.